### DISCUSSION TITLE
Problem with building when using Next v 13.1.4, failed to load next.config.js

### DISCUSSION THREAD

**Discussion by sroussey** [vercel/next.js#45140] [Help]
I only have this issue with my own repo, and can not reproduce on a small test one, so no issue to report yet.

```
npm run build
> next build

info  - Loaded env from /Users/steve/Code/embarc/.env.production.local
info  - Loaded env from /Users/steve/Code/embarc/.env
error - Failed to load next.config.js, see more info here https://nextjs.org/docs/messages/next-config-error

> Build error occurred
Error: Cannot find module 'ajv/dist/compile/codegen'
```

This is definitely related to npm and dependencies. If I have my repo back at next@13.1.3 it worked fine. But, and this is odd, if I **update** to next@13.1.33 it is also fine! But I if clean up with `rm -rf package-lock.json node_modules .next` and then `npm i && npm run build` I get the error above. Doing this cleanup on next@13.1.3 results in no errors.

If others start to hit this issue, there is a workaround: `npm i ajv@latest --save-dev`. Yes, it is very unsatisfying. But it is effective. I'll keep playing around with a new repo to see if I can get it to reproduce there as well.

**Comment by icyJoseph** [comment]
I made a new create-next-app and directly run `next build` after it installed its dependencies, and it went fine. 

I guess there's some package in your project that's breaking for unknown reasons... 

The change diff also doesn't seem to show anything related to ajv: https://github.com/vercel/next.js/compare/v13.1.3...v13.1.4
